interface/src/app/sample_annotation/annotation.spec.js

```diff
@@ -0,0 +1,68 @@
+describe('sample_annotation', function() {
+  var $httpBackend, $log, SampleMocks;
+
+  beforeEach(module('adage.sampleAnnotation'));
+  beforeEach(module('adage.mocks.sample'));
+  beforeEach(inject(function(_$httpBackend_, _$log_, _SampleMocks_) {
+    $httpBackend = _$httpBackend_;
+    $log = _$log_;
+    SampleMocks = _SampleMocks_;
+  }));
+
+  // Ensure no outstanding expectation or request at the end.
+  afterEach(function() {
+    $httpBackend.verifyNoOutstandingExpectation();
+    $httpBackend.verifyNoOutstandingRequest();
+    // very useful $log capture information from:
+    // http://www.jvandemo.com/how-to-access-angular-log-debug-messages-from-within-karma/
+    if ($log.debug.logs.length > 0) {
+      console.log($log.debug.logs);
+    }
+  });
+
+  describe('SampleAnnotationCtrl', function() {
+    var SampleAnnotationCtrl, $location, $scope, mockStateParams, $log;
```

Comment:
Good catch. I will clean this up a bit more. There's a bunch in there that's left over from the testing boilerplate that came with `ngBoilerplate` and is no longer used. I got a bunch out when ESLint found variables that were defined but not used, but these must have escaped its parser.

- [ ] removed unused testing artifacts